GroupIS1024/Pages/Index.cshtml

```diff
@@ -3,12 +3,14 @@
 @{
     ViewData["Title"] = "FantasyFootball";
 }
-<html>
```

Comment:
Added Doctype tag, and lang attribute to follow HTML best practices

Source: https://www.w3schools.com/html/html5_syntax.asp

Reply:
While you're here, also add XHTML 1.0 Strict validation tag.